lib/dice_bag/command.rb

```diff
@@ -25,15 +32,14 @@ def write(template_name, params = {})
     end
 
 
-    def generate_all_templates
+    def generate_all_templates(force=false)
       AvailableTemplates.all.each do |template|
-        generate_template(template)
+        generate_template(template, force)
       end
     end
 
-    def generate_template(default_template)
-      default_template.assert_existence
-      default_template.create_file
+    def generate_template(default_template, force=false)
+      copy_file default_template.file, default_template.destination, force: force
```

Comment:
it prints the action performed on specific files, e.g.
if new file created: 
```bash
create  config/dalli.yml.dice
```
if modified: 
```bash
force  config/dalli.yml.dice
```
if no changes: 
```bash
identical  config/dalli.yml.dice
```